hack/testing-olm/utils

```diff
@@ -193,3 +193,44 @@ EOL
 fi
   wait_for_deployment_to_be_ready $ns $name $((2 * $minute))
 }
+
+get_github_directory() {
+    local outputdir=$1
+    local ghdir=$2
+    local comp=$3
+    local repo=${4:-openshift}
+    local branch=${5:-master}
+    local name
+    local type
```

Comment:
type is a bash built-in, can we use another name please?

Reply:
fixed.  this was added from whence it was already commited